infoschema/metric_schema_test.go

```diff
@@ -37,12 +37,6 @@ func (s *inspectionSuite) TestMetricSchemaDef(c *C) {
 		} else {
 			c.Assert(def.Quantile == 0, IsTrue, Commentf("metric table %v has quantile, but doesn't contain $QUANTILE in promQL ", name))
 		}
-		if strings.Contains(def.PromQL, "$LABEL_CONDITIONS") {
-			c.Assert(len(def.Labels) > 0, IsTrue, Commentf("the labels of metric table %v should not be nil", name))
-		} else {
-			c.Assert(len(def.Labels) == 0, IsTrue, Commentf("metric table %v has labels, but doesn't contain $LABEL_CONDITIONS in promQL", name))
-		}
-
```

Comment:
No testcases these newly added metrics? At least check whether the syntax is right.

Reply:
The removed test is not for the `syntax` check. 
`syntax` test needs to use `Prometheus` parser. So currently I only use manual tests to check the syntax.

Reply:
I know how to test it now, no need to remove this test.